### PR TITLE
PULSE-224 add dob, ssn and tele

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: data-model.sql [hunk 1]
```diff
@@ -163,8 +163,8 @@ CREATE TABLE audit_request_source (
 	alternative_user_id varchar(100), --the process ID as used within the local operating system in the local system logs
 	user_name varchar(100),
 	user_is_requestor boolean default true,
-	role_id_code varchar(100), -- EV(110153, DCM, “Source”)
-	network_access_point_type_code smallint, --“1” for machine (DNS) name, “2” for IP address
+	role_id_code varchar(100), -- EV(110153, DCM, â€œSourceâ€�)
```
Comment (andlar) [inline data-model.sql]: Ack! Smart Quotes! Please replace all of these with "Real Quotes"!

File: data-model.sql [hunk 2]
```diff
@@ -378,8 +378,8 @@ CREATE TABLE patient (
 	id bigserial not null,
 	full_name varchar(255) not null,
 	friendly_name varchar(128) not null,
-	dob date,
-	ssn varchar(15),
+	dob varchar(100),
+	ssn varchar(9),
```
Comment (andlar) [inline data-model.sql]: On the off chance a person types in a SSN with dashes, will that fit in 9 characters?

File: data-model.sql [hunk 3]
```diff
@@ -471,7 +471,7 @@ ALTER TABLE query_organization OWNER to pulse;
 
 CREATE TABLE patient_record (
 	id bigserial not null,
-	dob date,
+	dob varchar(100),
 	ssn varchar(15),
```
Comment (andlar) [inline data-model.sql:475]: This SSN is a different length than the one for Patient; I think they should both be the same (and likely 15)

**Review by andlar — CHANGES_REQUESTED** [review]

**Review by kekey1 — COMMENTED** [review]
Nothing to add!

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**